.github/workflows/ci.yml

```
@@ -21,10 +21,28 @@ jobs:
       - run: npm test
       - run: git fetch --unshallow
         if: ${{ matrix.node-version == '16' && matrix.os == 'ubuntu-latest' }}
+      # Upload coverage for sonarcube (only matching OS and one node version required)
+      - uses: actions/upload-artifact@v3
+        if: ${{ matrix.os == 'ubuntu-latest' && matrix.node-version == '16' }}
```

```suggestion
        if: ${{ matrix.node-version == '16' && matrix.os == 'ubuntu-latest' }}
```

For consistency with line 23.

---

oooh, right, sorry! totally missed that as moving down to a separate step 😅